src/Sulu/Component/Contact/Tests/Unit/SmartContent/AccountDataProviderTest.php

```diff
@@ -222,14 +218,8 @@ public function testResolveDataSource(): void
     /**
      * @return DataProviderRepositoryInterface
      */
-    private function getRepository(
-        $filters = [],
-        $page = null,
-        $pageSize = 0,
-        $limit = null,
-        $result = [],
-        $options = []
-    ) {
+    private function getRepository($filters = [], $page = null, $pageSize = 0, $limit = null, $result = [], $options = [])
```

Comment:
looks like just an unexpected change so would keep it as it is else test method changes should be done on 2.5 branch if possible and non breaking changes